src/csharp/build_packages_dotnetcli.sh

```diff
@@ -38,8 +38,8 @@ dotnet pack --configuration Release Grpc.Core.Testing --output ../../../artifact
 dotnet pack --configuration Release Grpc.Auth --output ../../../artifacts
 dotnet pack --configuration Release Grpc.HealthCheck --output ../../../artifacts
 dotnet pack --configuration Release Grpc.Reflection --output ../../../artifacts
+dotnet pack --configuration Release Grpc.Tools --output ../../../artifacts
```

Comment:
looks like it's not possible to build  Grpc.Tools  on linux because of the net40 dependency. So should we remove it from here and add a comment/TODO mentioning that instead?

Reply:
Unfortunately, that does not quite work. I cannot compile the common Version.cs in Grpc.Tools, as it uses constants defined in Grpc.Core.dll, and task assemblies cannot have dependencies.

I had to copy the whole fix stanza to the tools project. Ugly as it is, I cannot think of a better solution. I'll refactor all projects/imports next, if it sounds good to you. There is indeed a lot of things that can be cleaned up.